frontend/src/metabase/actions/components/ActionForm/ActionFormFieldWidget.tsx

```diff
@@ -41,5 +41,5 @@ export const FormFieldWidget = forwardRef(function FormFieldWidget(
   const Widget =
     (formField.type ? WIDGETS[formField.type] : FormInputWidget) ??
     FormInputWidget;
-  return <Widget {...formField} nullable={formField.optional} ref={ref} />;
+  return <Widget {...formField} nullable ref={ref} />;
```

Comment:
`nullable` means that empty values are converted to `null`. It's different to a field being required/optional.

Reply:
@iethree I remember some shenanigans about nullable/optional action parameters, mind double-checking this?

Reply:
https://metaboat.slack.com/archives/C04FG88HL95/p1678369078943209

Reply:
long story short, we had to change the backend to be able to always send `null` for empty parameter values